blackbird_python/blackbird/program.py

```diff
@@ -98,6 +98,7 @@ def __init__(self, name="blackbird_program", version="1.0"):
         self._name = name
         self._version = version
         self._target = {"name": None, "options": dict()}
+        self._type = {"name": None, "options": dict()}
```

Comment:
Out of curiosity, do we want the default to be not None? I can't think of any good descriptor though. 'standard'?

Reply:
Oh, sorry @josh146. I overlooked this comment. I think that `None` could work, in the sense that this would just mean using the default type, but yes, having a name for the standard program might be a good idea. :thinking: